Assets/Mirror/Components/NetworkAnimator.cs

```diff
@@ -27,10 +28,10 @@ public class NetworkAnimator : NetworkBehaviour
         /// <summary>
         /// The animator component to synchronize.
         /// </summary>
-        [FormerlySerializedAs("m_Animator")]
+        [FormerlySerializedAs("animator")]
```

Comment:
No reason to remove the original FormerlySerializedAs...Unity handled it fine with no error.
```suggestion
        [FormerlySerializedAs("m_Animator")]
        [FormerlySerializedAs("animator")]
```

Reply:
`m_Animator` was changed about a year ago, it is unlikely we still need it

Reply:
it's from UNet, that's why it's there.

Reply:
In order to use NetworkAnimator  they would have to remove the UNet one, or go into the debug inspector and change the script field, in which case they know what they are doing.

I don't think we should be keeping obsolete things just because they were in UNet